<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApesOptions</class>
 <widget class="QDialog" name="ApesOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::West</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_general">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="hl_3">
         <item>
          <widget class="QLabel" name="lbl_verbosity">
           <property name="text">
            <string>Output verbosity</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cb_verbosity">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>Minimal</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Regular</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Full</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_plots">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>Plots</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Ignore lower than</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="dsb_ignorelwr">
           <property name="suffix">
            <string>%</string>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>0.200000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Unlabel lower than</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="dsb_unlabellwr">
           <property name="suffix">
            <string>%</string>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>5.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Line thickness</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sb_linethick">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>pts</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hl_2">
         <item>
          <widget class="QLabel" name="lbl_background">
           <property name="text">
            <string>Background color</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cb_background">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>Transparent (white)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Blue</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Green</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Red</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Cyan</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Magenta</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Yellow</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Black</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>White</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Line style</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cb_line_style">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Color scheme</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cb_color_scheme">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Error style</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="sb_error_style">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="title">
          <string>Plot Margins</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>left</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spinBox_3">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>right</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QSpinBox" name="spinBox_4">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>wspace</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QSpinBox" name="spinBox_5">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>top</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinBox_6">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>bottom</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QSpinBox" name="spinBox_7">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>hspace</string>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QSpinBox" name="spinBox_8">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_fitting">
      <attribute name="title">
       <string>Fitting</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_24">
           <property name="text">
            <string>Fitting Variable</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cb_fittingvar">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>Beta logarithm</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Beta</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_25">
           <property name="text">
            <string>Fitting Sequence</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cb_fittingseq">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>Best fit for all constants</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Remove worst beta in turn</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="gb_coarseoptions">
         <property name="title">
          <string>Coarse Fitting Criteria Options</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Threshold for χ²</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="le_coarsetolc">
              <property name="text">
               <string>1.0e-5</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Threshold for vars</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="le_coarsetolv">
              <property name="text">
               <string>1.0e-2</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_22">
              <property name="text">
               <string>Max. iterations</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="sb_coarsemaxits">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="maximum">
               <number>999</number>
              </property>
              <property name="value">
               <number>10</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gb_fineoptions">
         <property name="title">
          <string>Fine Fitting Criteria Options</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Threshold for χ²</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="le_finetolc">
              <property name="text">
               <string>1.0e-15</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Threshold for vars</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="le_finetolv">
              <property name="text">
               <string>1.0e-5</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_26">
              <property name="text">
               <string>Max. iterations</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="sb_finemaxits">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="maximum">
               <number>999</number>
              </property>
              <property name="value">
               <number>50</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_conc">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>Conc.</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Engine:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cb_engine">
           <property name="currentIndex">
            <number>1</number>
           </property>
           <item>
            <property name="text">
             <string>Python</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Fortran</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_19">
         <item>
          <widget class="QLabel" name="label_52">
           <property name="text">
            <string>Convergence Threshold</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_9">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_20">
         <item>
          <widget class="QLabel" name="label_53">
           <property name="text">
            <string>Max.Iterations</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_21">
         <item>
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>Interpolation</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cb_interp">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>Auto</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>None</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Linear</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Quadratic </string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Cubic</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_22">
         <item>
          <widget class="QLabel" name="label_51">
           <property name="text">
            <string>Forcer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cb_forcer">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>None</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Est</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Cubic</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Equil</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="2" column="0">
          <widget class="QCheckBox" name="cb_stepwise">
           <property name="text">
            <string>Stepwise fitting</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="cb_steplim">
           <property name="text">
            <string>Step limiter</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="cb_damping">
           <property name="text">
            <string>Damping</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="cb_scaling">
           <property name="text">
            <string>Scaling</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="cb_estimate">
           <property name="text">
            <string>Estimate initial guess</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QCheckBox" name="cb_zeroffd">
           <property name="text">
            <string>Zero off-diag</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>Warning: Do not change these options unless you know what you are doing.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hl_1">
     <item>
      <spacer name="hs_1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ApesOptions</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>529</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ApesOptions</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>529</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
